src/components/OSCALSystemImplementation.js

```diff
@@ -31,6 +31,187 @@ const OSCALSystemImplementationSubDataHeader = styled(TableCell)`
   padding: 0.75em 0.75em;
 `;
 
+const OSCALSystemImplementationTableTitle = styled(Typography)`
+  flex: 1 1 100%;
+`;
+
+function PropertiesTable(props) {
+  return (
+    <TableContainer>
+      <Table size="small">
+        <TableBody>
+          {props.list &&
+            props.list.map((property) => (
```

Comment:
```suggestion
          {props.list?.map((property) => (
```

I like the choice of using the optional chaining operator elsewhere. I would suggest using it here too and a couple other places.